src/mbgl/util/dtoa.cpp

```diff
@@ -1,10 +1,15 @@
 #include "dtoa.hpp"
 
+// Clang on Windows 64-bits can't parse rapidjson's dtoa
```

Comment:
I added a benchmark for comparing our `dtoa` against `std::to_string` and also a utest to assert the precision. `dtoa` is not only ~30 faster, but it offers higher precision.